R/bs-theme-preset.R

```diff
@@ -4,22 +4,27 @@ THEME_PRESET_CLASS <- "bs_theme_with_preset"
 resolve_bs_preset <- function(
   preset = NULL,
   bootswatch = NULL,
-  version = version_default()
+  version = NULL
 ) {
   if (is.null(preset) && is.null(bootswatch)) return(NULL)
-
+  
```

Comment:
I think we could do `version <- version %||% version_default()` early on here (so less changes are needed)?

Reply:
I'm also a little confused why `resolve_bs_preset()` needs to be changed at all?

Reply:
Yeah, great point, this was a holdover of the original approach in this PR, I'm pretty sure I can undo these changes entirely.